src/mars.d

```diff
@@ -66,6 +82,14 @@ extern (C++) const(char)* toWinPath(const(char)* src)
     return result;
 }
 
+
+/**
+ * Reads a file, terminate the program on error
+ *
+ * Params:
+ *   loc = The line number information from where the call originates
+ *   f = a `ddmd.root.file.File` handle to read
+ */
 extern (C++) void readFile(Loc loc, File* f)
```

Comment:
They should definitely get a new home, but I don't see how that should block them from being documented.